simple-weather/simple-weather/Controller/ViewController.swift

```diff
@@ -60,3 +66,18 @@ class ViewController: UIViewController, CLLocationManagerDelegate {
         }
     }
 }
+extension ViewController: UITableViewDelegate, UITableViewDataSource {
+    func numberOfSections(in tableView: UITableView) -> Int {
```

Comment:
You do not need to specify number of sections to return 1, because default value is 1, you can read about it in `UITableViewDataSource` `numberOfSections(in:)` [description](https://developer.apple.com/documentation/uikit/uitableviewdatasource/1614860-numberofsections). Only two methods are obligatory: `tableView(_:cellForRowAt:)` and `tableView(_:numberOfRowsInSection:)`